guides/source/active_record_basics.md

```diff
@@ -374,3 +374,86 @@ and to roll it back, `rake db:rollback`.
 Note that the above code is database-agnostic: it will run in MySQL,
 PostgreSQL, Oracle and others. You can learn more about migrations in the
 [Active Record Migrations guide](migrations.html).
+
+Enum
+----
+
+Active Record `enum` allows you declare an attribute where the values map to
+integers in the database, but can be queried by name.
+
+You can define the enum on a model calling the `enum` macro.
+
+```ruby
+class Post < ActiveRecord::Base
+  enum status: [ :active, :archived ]
+end
+```
+
+This will add `enum` on the column `status`, where the active is represented as 0
```

Comment:
What do you think about: 

> This will enable a mapping on the `status` column where `:active` is represented
> as 0 and `:archived` is represented as 1.